src/ioctls/vcpu.rs

```diff
@@ -1045,6 +1045,15 @@ impl VcpuFd {
         Ok(())
     }
 
+    /// Enables guest debugging and sets specific debug registers for x86
+    pub fn set_guest_debug(&self, debug_struct: &kvm_guest_debug) -> Result<()> {
```

Comment:
Can this ioctl be tested in any way? The coverage drops by 1%.

Reply:
Added a unittest for it